src/wallet/service.go

```diff
@@ -352,7 +352,12 @@ func (serv *Service) CreateAndSignTransaction(wltID string, password []byte, vld
 	}
 
 	if w.IsEncrypted() {
-		if err := w.guard(password, f); err != nil {
+		dw, err := w.unlock(password)
```

Comment:
why does guard need to be removed? guard doesn't need to relock anything

Reply:
The guard will lock the wallet at the end. I need to call defer `dw.erase()` here.

Reply:
guard:

unlock wallet, creates new wallet copy with unencrypted data